onboardingCli.js

```diff
@@ -0,0 +1,182 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
```

Comment:
I don't think so, it was in the base cli that I added to.

---
In reply to: [78635155](https://github.com/openT2T/opent2t-cli/pull/9#discussion_r78635155) [](ancestors = 78635155)